packages/playground-ui/src/ds/components/Shimmer/shimmer.css

```
@@ -1,26 +1,50 @@
 .shimmer-text {
-  --shimmer-period: 120px;
+  --shimmer-travel: 300%;
   --shimmer-dim: color-mix(in oklab, currentcolor 55%, transparent);
 
-  background-image: linear-gradient(90deg, var(--shimmer-dim) 0%, currentcolor 50%, var(--shimmer-dim) 100%);
-  background-size: var(--shimmer-period) 100%;
-  background-repeat: repeat;
+  background-image: linear-gradient(90deg, var(--shimmer-dim) 42%, currentcolor 50%, var(--shimmer-dim) 58%);
+  background-size: var(--shimmer-travel) 100%;
+  background-repeat: no-repeat;
   /* Chrome < 120 and Safari < 14 only clip to text prefixed. */
```

Stylelint (17.14.0)

[error] Expected empty line before comment (comment-empty-line-before)

(comment-empty-line-before)